ament_cmake_target_dependencies/cmake/ament_get_recursive_properties.cmake

```diff
@@ -0,0 +1,76 @@
+# Copyright 2020 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#
+# Get the recrusive include directories and libraries from interface targets.
```

Comment:
```suggestion
# Get the recursive include directories and libraries from interface targets.
```

Reply:
Fixed in 22f61014b11edfed656a4cc7f1c35f9be273dc3a.